packages/__docs__/src/NavToggle/index.tsx

```diff
@@ -23,32 +23,29 @@
  */
 
 import React, { Component } from 'react'
-import PropTypes from 'prop-types'
 
 import { ToggleDetails } from '@instructure/ui-toggle-details'
 import { Text } from '@instructure/ui-text'
 import { View } from '@instructure/ui-view'
 import { InstUISettingsProvider } from '@instructure/emotion'
 import { instructure } from '@instructure/ui-themes'
-
-class NavToggle extends Component {
-  static propTypes = {
-    summary: PropTypes.string.isRequired,
-    variant: PropTypes.oneOf(['section', 'category']),
-    children: PropTypes.node
-  }
-
+import type { NavToggleProps } from './props'
+import { propTypes, allowedProps } from './props'
+class NavToggle extends Component<NavToggleProps> {
```

Comment:
Please leave an empty line above the class declaration for better readability :)